### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: res/controllers/Hercules-DJ-Console-RMX-scripts.js [hunk 1]
```diff
@@ -909,7 +909,7 @@ HerculesRMX.rateChange = function (value, group) {
 //   HerculesRMX.stopJog[group] = 1.0;
 //   HerculesRMX.decayLast = new Date().getTime();
 //   engine.setValue(group,"scratch",HerculesRMX.stopJog[group]);
-//   HerculesRMX.stoping = true;
+//   HerculesRMX.stopping = true;
```
Comment (luzpaz) [inline res/controllers/Hercules-DJ-Console-RMX-scripts.js]: i couldn't find `.stoping` in the rest of the source or `.stopping` for that matter either. I just corrected this in order to ask what to do about this.

File: src/test/sampleutiltest.cpp [hunk 2]
```diff
@@ -488,7 +488,7 @@ BENCHMARK(BM_SampleUtilCopy)->Range(64, 4096);
 TEST_F(SampleUtilTest, copy3WithGainSpeed) {
     CSAMPLE* buffer = buffers[0];
 
-    int size = sizes[0] - (rand() % 2) * 8; // preven predicting loop size
+    int size = sizes[0] - (rand() % 2) * 8; // prevent predicting loop size
```
Comment (luzpaz) [inline src/test/sampleutiltest.cpp]: is this correct ?

**Comment by luzpaz** [comment]
Also there is another curious typo `sensivity` in the source. I was wondering if this was done purposely or is a typo of `sensitivity`. Search for it case-insensitively.

File: src/engine/sync/syncable.h [hunk 3]
```diff
@@ -93,7 +93,7 @@ class SyncableListener {
     // Notify Syncable that the Syncable's scratching state changed.
     virtual void notifyScratching(Syncable* pSyncable, bool scratching) = 0;
 
-    // A Syncable must never call notifyBeatDistanceChanged in respnse to a
+    // A Syncable must never call notifyBeatDistanceChanged in respnose to a
```
Comment (sblaisot) [inline src/engine/sync/syncable.h]: replace a typo with a typo :D

File: res/controllers/Hercules-DJ-Console-RMX-scripts.js (hunk 1 above)
Comment (sblaisot) [reply res/controllers/Hercules-DJ-Console-RMX-scripts.js]: this is just a comment (`//`) so correcting it should be no problem.

File: src/test/sampleutiltest.cpp (hunk 2 above)
Comment (sblaisot) [reply src/test/sampleutiltest.cpp]: yes, I think.

**Review by sblaisot — CHANGES_REQUESTED** [review]
Thanks you very much for taking care of this.
I added some minor comments. otherwise LGTM when you have applied the requested changes.

File: src/engine/sync/syncable.h (hunk 3 above)
Comment (luzpaz) [reply src/engine/sync/syncable.h]: :man_facepalming:

**Comment by luzpaz** [comment]
> Also there is another curious typo sensivity in the source. I was wondering if this was done purposely or is a typo of sensitivity. Search for it case-insensitively.

@sblaisot  please execute `grep -iR sensivity` to see what I mean

**Comment by sblaisot** [comment]
yes, seen. it should be safe to change all `sensivity` by `sensitivity` in controller mapping javascript.

**Comment by luzpaz** [comment]
> it should be safe to change all sensivity by sensitivity in controller mapping javascript.

Done in 6e455a6

**Comment by luzpaz** [comment]
Rebased and ready to go.

**Pull request merged**